test/test_arraycontext.py

```diff
@@ -180,8 +180,10 @@ def assert_close_to_numpy_in_containers(actx, op, args):
             ("where", 3),
             ("conj", 1),
             ])
-def test_array_context_np_workalike(actx_factory, sym_name, n_args):
-    actx = actx_factory()
+@pytest.mark.parametrize("force_device_scalars", [True, False])
+def test_array_context_np_workalike(actx_factory, sym_name, n_args,
+        force_device_scalars):
+    actx = actx_factory(force_device_scalars)
```

Comment:
This isn't ideal IMO; it changes the actx factory "interface" in an implementation-specific way. I assume that's what #44 is about.

Reply:
Yep, it's merged in now!